char-sheet/src/components/EquipmentTable.tsx

```diff
@@ -60,7 +61,25 @@ function EquipmentTable({equipment, container, service, editable=EditMode.Live}:
           items.map((item, i) => {
             return <tr className='hover tooltip tooltip-left w-full' data-tip={lookupDescription(item.name)} key={i}>
               <td className='w-full text-left'>
-                { item.name }
+                {item.custom ? (
```

Comment:
This whole logic block is a little gnarly. It might be fine, but it might also be easier to move it to a new helper component. Eh. 50/50.